java/eks/README.md

```diff
@@ -0,0 +1,32 @@
+# CDK Java Example
+
+This is an example of a CDK program written in Java.
+
```

Comment:
This should introduce

* What the sample intends to show
* How to deploy the sample
* What services the sample is interacting with
* How the customer can verify that the deployment is successful